### DISCUSSION TITLE
초기 테스트 상태

### DISCUSSION THREAD

**Discussion by dhihm** [huijeong-kim/ministore#8] [Q&A] [answered]
전 현재 리비전 기준으로 초기 상태가 이렇게 나옵니다. 

이 상태여도 진행하는데 무리는 전혀 없어 보이긴 하는데, 초기 의도한 거랑 동일한 상태인지 궁금해서요. 
이 부분. 
"test block_device_common::tests::all_block_device_type_should_tell_if_it_is_sync_or_async ... ok"

<img width="1017" alt="image" src="https://user-images.githubusercontent.com/28741816/235389460-076bcac3-35eb-446f-b244-5744b650c54d.png">

<img width="1035" alt="image" src="https://user-images.githubusercontent.com/28741816/235389436-d1b88d1b-4de9-4ba5-8b90-66510df81d4b.png">

**Comment by huijeong-kim** [comment, marked as answer]
네! 성공한 두 개의 테스트는 제공하는 코드(?) 에 대한 테스트여서 성공으로 뒀어요. 말씀하신 테스트는 디바이스 타입 추가할 때 마다 모두 통과하길 기대합니다 (테스트 돌리기 위한 필수 조건)

**Reply by dhihm** [reply]
그렇군요 초기 가이드 문서를 한 번 업뎃해보겠습니다 ㅋㅋ

**Comment by dhihm** [comment]
메시지를 좀 더 자세히 보니까 중간에 panic이 나는 것 같네요. 좀 더 살펴 볼께요.
